packages/next-auth/src/providers/shopify.js

```diff
@@ -0,0 +1,46 @@
+export default function Shopify(options) {
+  const { shop, apiVersion = "2022-01" } = options
+
+  return {
+    id: "shopify",
+    name: "Shopify",
+    type: "oauth",
+    authorization: {
+      url: `https://${shop}.myshopify.com/admin/oauth/authorize`,
```

Comment:
I much preferred if instead of adding a new provider-specific config option `shop`, we required `issuer` to be set to the URL `https://${shop}.myshopify.com` to align with other providers.

Reply:
Are you recommending renaming `shop` to `issuer` here basically?